pkg/dogstatsd/server.go

```diff
@@ -64,6 +70,13 @@ func init() {
 	dogstatsdExpvars.Set("UnterminatedMetricErrors", &dogstatsdUnterminatedMetricErrors)
 }
 
+// used in debug mode to add the origin on the processed metric as a tag
+type cachedTagsOriginMap struct {
+	origin string
+	ok     map[string]string
+	err    map[string]string
```

Comment:
Could we just has one filed called `tags` ?

Reply:
They're not containing the same entries (for the `state` key).

Reply:
ho yeah, I see.